### ISSUE TITLE
Request feature for yarn t to act like npm t --shortcut for yarn test 

### ISSUE THREAD

**Issue by troy0820** [yarnpkg/yarn#902]
### Requesting feature for shortcut `yarn t`

I know that there are other issues that are requesting aliases and shortcuts for certain commands.  

I think we are all used to using npm and trying to use yarn as a drop in replacement.  

This issue is a request for a feature for `yarn t`  to act like `npm t`

**Comment by pi0** [comment]
Please checkout [YarNPM](https://github.com/pi0/yarnpm) :) **WIP**

**Comment by busches** [comment]
@kittens based on this [comment](https://github.com/yarnpkg/yarn/issues/1257#issuecomment-255942428) and others, this doesn't appear to be something the team wants to support.

**Comment by voxsim** [comment]
@bestander this seems a feature request, not a triage, do we want support this?

**Comment by bestander** [comment]
@voxsim, feature requests we mark as "triaged" and "new-feature" and "discussion".

**Comment by bestander** [comment]
This one was discussed before.
As of now Yarn does not add shortcuts to commands, users can add scripts "t" to their package.json to achieve the same thing

**Comment by pi0** [comment]
@bestander sorry if asking in wrong thread but just didn't wanted to open another unrelated issue :)) Why yarn team decided to replace `install` with `add` ? Is there any topic about that ?

**Comment by bestander** [comment]
Yeah, I think it was discussed in the earlier issues.
We need to write a blog post about it :)

tl;dr `install` is a command that generates node_modules based on project's manifest and `add/upgrade/remove` are commands that change your manifest.
They have different semantics and intentions
